src/Admin/AbstractAdmin.php

```diff
@@ -1735,6 +1761,14 @@ public function addChild(AdminInterface $child)
         $this->children[$child->getCode()] = $child;
 
         $child->setParent($this);
+
+        // TODO: remove $args and add $field parameter to this function on next Major
```

Comment:
we use NEXT_MAJOR instead of TODO normally

Reply:
my bad, not sure where did TODO come from